airflow/executors/kubernetes_executor.py

```diff
@@ -548,11 +548,11 @@ def execute_async(
         executor_config: Any | None = None,
     ) -> None:
         """Executes task asynchronously"""
-        self.log.info('Add task %s with command %s with executor_config %s', key, command, executor_config)
```

Comment:
Can we add the executor_config at the debug level at least? There are cases where you want to see the config that aren't full-on invalid.

Reply:
Fair enough, though if something is invalid it would fail and be logged in the exception block below? Bad pods would also be logged on line 270 once create_namespaced_pod fails.

Anyway, I will restore it but change to debug.

Reply:
Maybe one need to distinguish between different contents of the executor_config? We mainly use `pod_override` which get grossly verbose in logs. `pod_template_file` don't have that issue and is likely more of interest if someone is using such.
What type of invalid content are you concerned with? Maybe other validation logic should be added instead?

Reply:
I'm actually concerned mostly with "valid" content that is nonetheless troublesome to the task. E.g. asking for more memory than any node actually has or mounting a drive to the wrong location.

Reply:
Can't such issues be inspected via k8s? Dumping entire pod spec is almost akin to dumping entire dag-file. Trace everything feels a bit overkill imo. 
In either case, will restore this to debug once I'm at desk tomorrow.

Reply:
Done now.